### ISSUE TITLE
segfault on systemctl enable|is-enabled with zero-length unit arg

### ISSUE THREAD

**Issue by absurd34** [systemd/systemd#5483]
### Submission type

  - [x] Bug report
  - [ ] Request for enhancement (RFE)

*NOTE: Do not submit anything other than bug reports or RFEs via the issue tracker!*

### systemd version the issue has been seen with

> 230, 232

*NOTE: Do not submit bug reports about anything but the two most recently released systemd versions upstream!*

### Used distribution

> Debian

### In case of bug report: Expected behaviour you didn't see

> Some invalid argument error handling.

### In case of bug report: Unexpected behaviour you saw

> segfault

### In case of bug report: Steps to reproduce the problem

> systemctl enable ''
> systemctl is-enabled ''

fwiw, I get this on a current Debian sid (systemd 232):

```
$ absurd? systemctl enable ''
*** Error in `systemctl': free(): invalid pointer: 0x00007f5f19beeb58 ***
======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(+0x70bcb)[0x7f5f198c6bcb]
/lib/x86_64-linux-gnu/libc.so.6(+0x76f96)[0x7f5f198ccf96]
/lib/x86_64-linux-gnu/libc.so.6(+0x7778e)[0x7f5f198cd78e]
/lib/systemd/libsystemd-shared-232.so(strv_clear+0x29)[0x7f5f19f0a9b9]
/lib/systemd/libsystemd-shared-232.so(strv_free+0x9)[0x7f5f19efa839]
systemctl(+0x15b95)[0x7f5f1a46eb95]
systemctl(+0x164f0)[0x7f5f1a46f4f0]
/lib/systemd/libsystemd-shared-232.so(dispatch_verb+0x20d)[0x7f5f19ef101d]
systemctl(+0x7d5b)[0x7f5f1a460d5b]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf1)[0x7f5f198762b1]
systemctl(+0x99ba)[0x7f5f1a4629ba]
```

Thx!

S

**Comment by poettering** [comment]
Fix waiting in #5485

**Comment by absurd34** [comment]
Thx Lennart!
